android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/Utils.java

```diff
@@ -54,6 +59,34 @@ private static void throwEncodeException(JSONException e) {
     throw new RuntimeException("Could not encode JSON", e);
   }
 
+  private static String getApiKey(Context context){
+    int resourceId = context.getResources().getIdentifier("google_api_key", "string",
+                                                          context.getPackageName());
+    return resourceId != 0 ? context.getString(resourceId) : "";
+  }
+
+  public static void setPwsEndpoint(Context context, PwsClient pwsClient){
```

Comment:
This will be changed once the settings get added, so I am inclined to leave the generic name.